_episodes/01-why-bother-performance.md

```diff
@@ -141,14 +136,14 @@ The `--time` variable used in the exercise is the amount of CPU hours requested,
 > 3. Optimising the code
 > 4. Having a computer with higher flops
 > 
-> > ## Solution
-> > 1. Yes, the more cores and nodes you have, the more work can be distributed across those cores.
-> > 2. No, increasing simulation walltime and CPUh only increases the duration the code will run for. It does not improve the code performance.
-> > 3. Yes, making the code more efficient by cutting out unnecessary loops and bottlenecks can greatly increase performance.
-> > 4. Yes, the faster the computer, the faster the code can run, however, this may not always be possible logistically.
-> >
-> > As you can see, there are a lot of right answers, however some methods work better than others, and it can entirely depend on the problem you are trying to solve.
-> >
+>> ## Solution
+>> 1. Yes, the more cores and nodes you have, the more work can be distributed across those cores.
+>> 2. No, increasing simulation walltime and CPUh only increases the duration the code will run for. It does not improve the code performance.
+>> 3. Yes, making the code more efficient by cutting out unnecessary loops and bottlenecks can greatly increase performance.
+>> 4. Yes, the faster the computer, the faster the code can run, however, this may not always be possible logistically.
```

Comment:
Historically this was what happened but modern HPC machines worry a lot about their electricity bill. Slower CPUs, but lots of them, is a more power-efficient way to achieve higher *overall* performance.

Reply:
Another caveat here is that sometimes it is not the speed of the CPU, but how fast you can get the data it needs to it. If your CPU spends time idle waiting for data to arrive (*I/O bound* calculations), a faster CPU does not help.